src/main/java/in/ashwanthkumar/gocd/github/util/GitCmdHelperExtended.java

```diff
@@ -0,0 +1,26 @@
+package in.ashwanthkumar.gocd.github.util;
+
+import java.io.File;
+
+import com.tw.go.plugin.cmd.Console;
+import com.tw.go.plugin.cmd.ProcessOutputStreamConsumer;
+import com.tw.go.plugin.git.GitCmdHelper;
+import com.tw.go.plugin.model.GitConfig;
+import org.apache.commons.exec.CommandLine;
+
+public class GitCmdHelperExtended extends GitCmdHelper {
```

Comment:
question (non-blocking): I do not know what the GitCmdHelper does and was wondering if the 'extended' could be replaced with a more precise name.

Reply:
`GitCmdHelper` provides a set of methods to run Git commands (clone, reset, etc). It's not exhaustive, so I had to add this extended helper with an additional command (checking out a new branch).

I can't really think of a more precise name, because the aim of this class is to provide arbitrary additional commands (even if it's only the checkout command at the moment).
But I think I'll change the word order to `ExtendedGitCmdHelper` because it reads a bit better and follows the pattern of `XYZHelper`.